projects/auth0-angular/src/lib/auth.module.ts

```diff
@@ -17,7 +17,12 @@ export class AuthModule {
       providers: [
         AuthService,
         AuthGuard,
-        { provide: AuthConfigService, useFactory: () => config },
+        {
+          provide: AuthConfigService,
+          useFactory(): AuthConfig {
+            return config;
+          },
```

Comment:
Shouldnt we just use `useValue` here ? This doesnt look like a factory.

Reply:
You're right. I remember when I did this I was getting issues when using a null value with `useValue`, in cases where the config wasn't supplied to `forRoot`, but there must have been something else getting in the way. I've tested it now with `useValue` and appears to work fine, so I've updated the code. Thanks!